react-app-drawer-component/my-app/src/AppDrawer.js

```diff
@@ -0,0 +1,39 @@
+import './AppDrawer.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBars } from '@fortawesome/free-solid-svg-icons';
+
+export default function AppDrawer({
+  menuClicked,
```

Comment:
It's best to name your props that are functions to start with "on". That makes it more clear they are used to respond to events. The functions that are passed to these props often start with "handle", but the props themselves should start with "on".

So, onClicked, onIcon, onMenu, for example.